root/static/scripts/common/components/ReleaseEvents.js

```diff
@@ -123,11 +125,14 @@ const ReleaseEvents = ({
                   title={l('Show all release events')}
                 >
                   {bracketedText(texp.l('show {n} more', {
-                    n: events.length - COLLAPSE_THRESHOLD,
+                    n: events.length - (TO_SHOW_BEFORE + TO_SHOW_AFTER),
                   }))}
                 </a>
               </li>
-              {buildReleaseEventRow(events[events.length - 1], abbreviated)}
+              {buildReleaseEventRow(
+                events[events.length - TO_SHOW_AFTER],
+                abbreviated,
+              )}
```

Comment:
This needs to be changed to use a slice like `TO_SHOW_BEFORE` has, 'cause right now this does/means something different (an offset of a single item from the end)

Reply:
I think it should be fine now!